### PR TITLE
[TRUS-3100] - Residency, Nationality and Legally incapable questions

### PR DESCRIPTION
**Pull request by jassalrichy**
Residency, Nationality and Legally incapable question for Individual Beneficiary in Trust Registration

### CODE REVIEW

File: app/pages/register/beneficiaries/individual/mld5/CountryOfNationalityInTheUkYesNoPage.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+final case class CountryOfNationalityInTheUkYesNoPage(index : Int) extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ Beneficiaries \ IndividualBeneficiaries \ index \ toString
+
+  override def toString: String = "countryOfNationalityInTheUkYesNo"
+
+  override def cleanup(value: Option[Boolean], userAnswers: UserAnswers): Try[UserAnswers] =
+    value match {
+      case Some(true) => userAnswers.set(CountryOfNationalityPage(index), "GB")
```
Comment (jackheslop96) [inline app/pages/register/beneficiaries/individual/mld5/CountryOfNationalityInTheUkYesNoPage.scala:34]: Very minor but could `"GB"` perhaps be stored in a Constants object if it is referenced in multiple places?

File: conf/messages.en [hunk 2]
```diff
@@ -177,6 +177,49 @@ individualBeneficiary.roleInCompany.NA = None of these
...
+individualBeneficiary.5mld.countryOfNationality.heading = What is {0}’s country of nationality?
+individualBeneficiary.5mld.countryOfNationality.checkYourAnswersLabel = What is {0}’s country of nationality?
+individualBeneficiary.5mld.countryOfNationality.error.required = Enter a country
+individualBeneficiary.5mld.countryOfNationality.error.length = The country of nationality must be 100 characters or less
+individualBeneficiary.5mld.countryOfNationality.error.invalidCharacters = The country of nationality must only include letters a to z, apostrophes, commas, full stops, hyphens, round brackets and spaces
+
+individualBeneficiary.5mld.legallyIncapableYesNo.title = Do you know if the individual beneficiary is legally incapable at the time of registration?
+individualBeneficiary.5mld.legallyIncapableYesNo.heading = Do you know if {0} is legally incapable at the time of registration?
+individualBeneficiary.5mld.legallyIncapableYesNo.checkYourAnswersLabel = Do you know if {0} is legally incapable at the time of registration?
+individualBeneficiary.5mld.legallyIncapableYesNo.error.required = Select yes if you know the beneficiary’s legally capacity
```
Comment (jackheslop96) [inline conf/messages.en:220]: Should this be `Select yes if you know the beneficiary’s **legal** capacity` ?

File: app/pages/register/beneficiaries/individual/mld5/CountryOfNationalityInTheUkYesNoPage.scala (hunk 1 above)
Comment (jassalrichy) [reply app/pages/register/beneficiaries/individual/mld5/CountryOfNationalityInTheUkYesNoPage.scala]: Updated

File: conf/messages.en (hunk 2 above)
Comment (jassalrichy) [reply conf/messages.en]: Agree, however the prototype has the error message:
"Select yes if you know the beneficiary's legally capacity"
Will apply "Select yes if you know the beneficiary's legal capacity"

**Pull request merged**